dcrjson/dcrwalletextcmds.go

```diff
@@ -124,6 +124,23 @@ type GetTicketsCmd struct {
 	IncludeImmature bool
 }
 
+// NewGetTicketsCmd creates a new GetTicketsCmd.
+func NewGetTicketsCmd(includeImmature bool) *GetTicketsCmd {
+	return &GetTicketsCmd{includeImmature}
+}
+
+// GetTicketVoteBitsCmd is a type handling custom marshaling and
+// unmarshaling of getticketvotebits JSON wallet extension
+// commands.
+type GetTicketVoteBitsCmd struct {
+	TxHash string
+}
+
+// NewGetTicketsVoteBitsCmd creates a new GetTicketsVoteBitsCmd.
```

Comment:
// NewGetTicketsVoteBitsCmd returns a new instance which can be used to issue a getticketsvotebits JSON-RPC command.